config/dictionaries/resource.json

```diff
@@ -592,10 +590,10 @@
     "id": "application_environment_variable",
     "name": "Fleet environment variable",
     "fields": [
-      "id",
-      "created_at",
-      "application",
-      "name",
+      "id", 
+      "created_at", 
+      "application", 
+      "name", 
```

Comment:
There seems to be trailing space on these lines.

Reply:
Trimed